### PR TITLE
[hotfix][docs] Correct reference link for DB2 docs

### PR DESCRIPTION
**Pull request by Jiabao-Sun**
Correct reference link for DB2 docs

### CODE REVIEW

File: docs/content/connectors/db2-cdc.md [hunk 1]
```diff
@@ -246,8 +246,7 @@ Data Type Mapping
 <table class="colwidths-auto docutils">
     <thead>
       <tr>
-        <th class="text-left" style="width:30%;">Db2 type<a href="https://dev.mysql.com/doc/man/8.0/en/data-types.
-html"></a></th>
+        <th class="text-left" style="width:30%;">Db2 type<a href="https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-data-types"></a></th>
```
Comment (rookiegao) [inline docs/content/connectors/db2-cdc.md:249]: Do users need to visit the link here？if necessary,Do you want to change it to the following form?

<th class="text-left" style="width:30%;"><a href="https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-data-types">Db2 type</a></th> 

The current writing method does not seem to generate any hyperlink.

Comment (Jiabao-Sun) [reply docs/content/connectors/db2-cdc.md]: Thanks @rookiegao to point this.
Fixed.

**Comment by rookiegao** [comment]
Thanks @Jiabao-Sun for this contribution. I left a litter comments.

image

On line 163 of the file

and "latest-offset". Please see <a href="#startup-reading-position">Startup Reading Position</a>section

A space is required here

besides, LGTM

**Comment by Jiabao-Sun** [comment]
Thanks @rookiegao for the detailed review. 
Fixed.

File: docs/content/connectors/db2-cdc.md [hunk 2]
```diff
@@ -246,8 +246,7 @@ Data Type Mapping
 <table class="colwidths-auto docutils">
     <thead>
       <tr>
-        <th class="text-left" style="width:30%;">Db2 type<a href="https://dev.mysql.com/doc/man/8.0/en/data-types.
-html"></a></th>
+        <th class="text-left" style="width:30%;"><a href="https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-data-types">Db2 type</a></th>
```
Comment (ruanhang1993) [inline docs/content/connectors/db2-cdc.md:249]: I think this is better to link to https://www.ibm.com/docs/en/db2/11.5?topic=elements-data-types.

**Review by ruanhang1993 — COMMENTED** [review]
Thanks for your work, @Jiabao-Sun. Just a little suggestion.

File: docs/content/connectors/db2-cdc.md (hunk 2 above)
Comment (Jiabao-Sun) [reply docs/content/connectors/db2-cdc.md]: Thanks @ruanhang1993 for this suggestion.
Fixed.

**Review by ruanhang1993 — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**